config/openhpc.yml

```diff
@@ -29,15 +29,15 @@ cluster_groups:
 slurm_login:
```

Comment:
This is (mainly) where p3-appliances falls apart for me. This file seems to get updated with whatever the current state of the cluster is (let's hope we never have more than one cluster...). I'm sitting on some thoughts about this and will try to put them together.

Reply:
Well, at the moment, we only have one cluster, but could make of copy of this config file if we have more than one. E.g. I've duplicated this config file for nvme server.